etc/docs/build.ts

```diff
@@ -69,18 +68,40 @@ async function copyNewDocsToGeneratedSite({ semverVersion }: VersionSchema) {
   return await exec(command);
 }
 
+function customSemverCompare(a: string, b: string) {
+  if (a === 'current') {
+    return -1
+  }
+  if (b === 'current') {
+    return 1
+  }
+  // put legacy 3x driver version at the bottom of the list
+  if (a === 'core') {
+    return 1;
+  }
+  if (b === 'core') {
+    return -1;
+  }
+
+  return b.localeCompare(a);
```

Comment:
Unfortunately the cheat method won't work here:

```js
['5.9', '5.10','5.11','10.5'].sort((a,b)=>b.localeCompare(a)) 
// [ '5.9', '5.11', '5.10', '10.5' ]
```

Reply:
We have `semver` as a dev dep, could we use that?

Reply:
The plain `compare` is not meant to work with invalid npm versions but there might be some special function functionality you could reuse? a string split and numeric compare should be pretty trivial though

Reply:
I think `semver.greaterThan` would work.  But now I'm not sure without testing it some 😅

Reply:
something like '5.1' (as opposed to something like '5.1.1' is not valid semver, so you can't put it directly into the comparators. Doing a map `split` chained with map `Number` on the inputs would be one line of code, a conditional comparing them piece by piece another line, so either which way, it's two lines of code. 😉